config.go

```diff
@@ -29,18 +27,15 @@ type Config struct {
 }
 
 const (
-	configDefaultBackendAPI             string = "https://backendapi.corbado.io"
-	configDefaultFrontendAPI            string = "https://%s.frontendapi.corbado.io"
-	configDefaultShortSessionCookieName string = "cbo_short_session"
-	configDefaultCacheMaxAge                   = time.Minute
+	configDefaultCacheMaxAge = time.Minute
 
 	configDefaultJWKSRefreshInterval  = time.Hour
 	configDefaultJWKSRefreshRateLimit = 5 * time.Minute
 	configDefaultJWKSRefreshTimeout   = 10 * time.Second
 )
 
 // NewConfig returns new config with sane defaults
-func NewConfig(projectID string, apiSecret string) (*Config, error) {
+func NewConfig(projectID string, apiSecret string, frontendAPI string, backendAPI string) (*Config, error) {
```

Comment:
shouldn't we have some sane defaults for BAPI and FAPI?
I get that FAPI should use the project CNAME, but BAPI should have a default value?

The less users have to provide by default the better

Reply:
we decided to go with "there are always 4 values needed for backend sdks: project id, api secret, fapi and bapi". fapi makes sense to practice because when you go on prod you need to provide the cname as fapi and bapi just to make it "complete" and might be needed in the future if we have multiple environments.